packages/components/src/components/ContentfulContainer/ContentfulContainerAsHyperlink.tsx

```diff
@@ -30,6 +30,7 @@ export type ContentfulContainerAsHyperlinkProps<EditorMode = boolean> = (EditorM
   cfHyperlink?: StyleProps['cfHyperlink'];
   cfOpenInNewTab?: StyleProps['cfOpenInNewTab'];
   WrapperComponent?: React.ElementType;
+  dragProps?: any; // TODO: add type
```

Comment:
```suggestion
  dragProps?: DragWrapperProps;
```
I saw you used the same prop below. Would that work if we moved the type into the core package?

Reply:
@Chaoste good catch! Went ahead and moved the types to core and now this works properly